pennylane/_qubit_device.py

```diff
@@ -2011,3 +2013,27 @@ def _adjoint_jacobian_processing(jac):
 
         # must be 2-dimensional
         return tuple(tuple(np.array(j_) for j_ in j) for j in jac)
+
+    def _get_diagonalizing_gates(self, circuit: QuantumTape):  # pylint:disable=no-self-use
```

Comment:
```suggestion
    def _get_diagonalizing_gates(self, circuit: QuantumTape) -> List[Operator]:  # pylint:disable=no-self-use
```

Reply:
not imported but i'll import and do this